babel.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker-compose exec web npm install babel-core@^7.0.0-bridge.0 --save-dev
```

Comment:
We shouldn't need a shell script to install this.  As above, once it's installed during development, it should be in the package.json.

Reply:
deleted line